.clang-format-ci

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+#
+# Copyright 2018-present The Material Components for iOS Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Fail on any error.
+set -e
+
+# The tagged version of https://github.com/material-foundation/clang-format-ci to check out.
+CLANG_FORMAT_CI_VERSION="v1.*"
```

Comment:
Is this a regex or the actual tag value?

Reply:
Clarified the docs.